packages/react/src/components/loading/LoadingCircle.tsx

```diff
@@ -0,0 +1,3 @@
+import {FunctionComponent} from 'react';
+
+export const LoadingCircle: FunctionComponent<any> = () => <div className={'loading-circle'}></div>;
```

Comment:
We should add an aria role, this way we won't need to rely on the class name in the tests